extension.js

```diff
@@ -7,6 +7,22 @@ const PanelMenu = imports.ui.panelMenu;
 const Me = ExtensionUtils.getCurrentExtension();
 const workspaceManager = global.workspace_manager;
 
+
+// workspace switch to previous / next
+let scroll_wrap = false;
+function workspace_switch(d){
+  let i = workspaceManager.get_active_workspace_index()
+  let n = workspaceManager.get_n_workspaces()
+
+  let x = (i+d+n)%n
+  if (!scroll_wrap){
+    if(i+d>=n || i+d<0) x = i
+  }
+
+  // print(`=== workspace_switch ${d}: ${i} => ${x}`)
```

Comment:
could you remove this commented out code please?

Reply:
I thought the log might help to understand and troubleshoot the execution process 

of course, I'll remove those log/print comment